lib/logflare/endpoints/cache.ex

```diff
@@ -154,8 +159,10 @@ defmodule Logflare.Endpoints.Cache do
   end
 
   def handle_cast(:touch, state) do
-    state = %{state | last_query_at: DateTime.utc_now()}
-    {:noreply, state}
+    :timer.cancel(state.shutdown_timer)
+    timer = state.query |> cache_duration_ms() |> shutdown()
+
+    {:noreply, %{state | shutdown_timer: timer, last_query_at: DateTime.utc_now()}}
```

Comment:
this changes the behaviour to extend the max lifetime of the cache by the cache duration on each endpoint query. Can result in stale data.
Also no test coverage for the touch behaviour change.
imo should only extend cache duration if proactive requerying is set

Reply:
removed the touch/1 api, we aren't using it anyways.